### PR TITLE
Update header to new mocks.

### LINKED ISSUE #74: Modify the header links to match the new mockup.
The orientation and size of the links in the header should match the following mockup:
https://www.figma.com/proto/5fgkTZjeBKML4GHHSfw5Hn/Mocks?node-id=5%3A1&scaling=min-zoom

### PR DESCRIPTION
**Pull request by turnerhayes**
Fixes #74.

### CODE REVIEW

File: client/components/Navigation.js [hunk 1]
```diff
@@ -15,15 +16,24 @@ import logoUrl from '../images/CityOfSanJose_logo.png';
...
-    {children}
-  </Link>
-);
+let NavLink = ({ location, path, children }) => {
+  const isActive = location.pathname === path;
+  const className = 'c_navigation__link';
+  return (
+    <Link
+      to={path}
+      className={classnames(
```
Comment (JMStudiosJoe) [inline client/components/Navigation.js:25]: can we clean this up a little? Took a minute to understand

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]

File: client/components/Navigation.js (hunk 1 above)
Comment (turnerhayes) [reply client/components/Navigation.js:25]: Can you elaborate a bit about what was difficult to understand? Is it the use of `classnames()`?

Comment (JMStudiosJoe) [reply client/components/Navigation.js:25]: its mainly just the nestedness when you could declare a const and then just apply. In my opinion, it just reads easier when reviewing code.

**Review by JMStudiosJoe — APPROVED** [review]
only code preference no reason for changes will approve and merge

**Pull request merged**